### ISSUE TITLE
Frame generation not active in some games with rx 5700xt card

### ISSUE THREAD

**Issue by MK-JH** [artur-graniszewski/DLSS-Enabler#13]
Hi, dear Arthur
Is there a solution to fix the problem of frame generation not being activated by selecting the dlss fg option in the god of war ragnarok game menu on the rx 5700xt card??
(I tested beta 8 and 9)

This problem is also present on Alan Wake 2 and frame generation is not activated on this card

There is no solution??
tnx

**Comment by artur-graniszewski** [comment]
Hi,

AlanWake2 has always been a problematic game, but mainly for NVIDIA GPUs (basically, after enabling the FG option game just duplicated the frames rather than generate the proper ones). It was curiously working properly on AMD and INTEL GPUs.

Please provide the dlss-enabler.log file from that game's directory so we can see what happened.

As for Ragnarok, we tried to troubleshoot that game two days ago and it seems that its.... broken, even for RTX 40XX cards. Hopefully upcoming patches will fix the issue (or already did since we last checked). Please make sure your game is up to date, and send the logs too, just to compare with our findings....

**Comment by MK-JH** [comment]
> Hi,
> 
> AlanWake2 has always been a problematic game, but mainly for NVIDIA GPUs (basically, after enabling the FG option game just duplicated the frames rather than generate the proper ones). It was curiously working properly on AMD and INTEL GPUs.
> 
> Please provide the dlss-enabler.log file from that game's directory so we can see what happened.
> 
> As for Ragnarok, we tried to troubleshoot that game two days ago and it seems that its.... broken, even for RTX 40XX cards. Hopefully upcoming patches will fix the issue (or already did since we last checked). Please make sure your game is up to date, and send the logs too, just to compare with our findings....
> 
> Also, out of curiosity, you can try the BETA10 build. Not sure if it will help (probably no), but one thing changed that helped to fix SpecialK tool. I wonder if the same issue impacted some games too (like AW2 or Ragnarok).

thank you
Today, giving an update and this problem of Dlss FG not being activated in God of War Ragnarok game was solved

There is still the problem of games crashing and freezing when using Latency flex, and it has not been solved. I tested the beta 10 build and after playing for a while, the game crashes or freezes completely, and we have to close the game and open it again.

**Comment by artur-graniszewski** [comment]
I hoped the soft-lock issue has been solved in BETA9 already. Could you play a bit longer using BETA 10 and ReflexEmulation=on to double check if its still an LFX issue?

**Comment by MK-JH** [comment]
> I hoped the soft-lock issue has been solved in BETA9 already. Could you play a bit longer using BETA 10 and ReflexEmulation=on to double check if its still an LFX issue?

I test and announce the result

**Comment by MK-JH** [comment]
> I hoped the soft-lock issue has been solved in BETA9 already. Could you play a bit longer using BETA 10 and ReflexEmulation=on to double check if its still an LFX issue?

Dear Arthur
I tested mode ReflexEmulation=on in beta build 10 and the crash and freeze problem was completely solved
But in the mode
ReflexEmulation=auto
After playing for a while, it crashes and freezes

**Comment by artur-graniszewski** [comment]
Thanks for checking this issue.

I have two requests to you if you don't mind:
* would you like to join our Discord channel to speed up the troubleshooting of LFX?
* could you share the dlss-enabler.log data here on github? it can be a log from a proper game run.

**Comment by MK-JH** [comment]
> Thanks for checking this issue.
> 
> I have two requests to you if you don't mind:
> 
> * would you like to join our Discord channel to speed up the troubleshooting of LFX?
> * could you share the dlss-enabler.log data here on github? it can be a log from a proper game run.

Yes, send me your invitation link

This file dlss-enabler.log in mode
ReflexEmulation=on
Beta version 10
god ragnarok game

[dlss-enabler.log](https://github.com/user-attachments/files/17090684/dlss-enabler.log)

**Comment by artur-graniszewski** [comment]
Thanks.

Here's the Discord invitation: https://discord.com/invite/2JDHx6kcXB

Please join the "DLSS Enabler Mod" channel.

I will be off today (starting in few minutes) but would be happy to get in touch with you tomorrow

**Comment by MK-JH** [comment]
> DLSS Enabler Mod

Unfortunately, I get this error

![Untitled](https://github.com/user-attachments/assets/a81edcca-d25d-406d-92f0-16835c649093)

**Comment by MK-JH** [comment]
> Thanks.
> 
> Here's the Discord invitation: https://discord.com/invite/2JDHx6kcXB
> 
> Please join the "DLSS Enabler Mod" channel.
> 
> I will be off today (starting in few minutes) but would be happy to get in touch with you tomorrow

Unfortunately, I get this error

![Untitled](https://github.com/user-attachments/assets/743189b5-6a67-47d1-8084-d0501d15c30e)

**Comment by artur-graniszewski** [comment]
Hi,

I'm trying to figure this out (never seen such issue). Are you using some kind of VPN? Are you behind some government firewall?

**Comment by MK-JH** [comment]
> Hi,
> 
> Did you go through verification process? (I assume you should receive an email with a call to action button inside)

Yes, all the steps have been completed and I can use other channels Without any problem, but I have a problem with your channel

Discord is filtered in my country and I force to use vpn

Anyway, my time is completely free and I can do any necessary test for you
Tell me here and I will do it for you

tnx

**Comment by artur-graniszewski** [comment]
Hi, just say "hi" on DLSS ENABLER MOD chat and we will start the testing;)
